apps/frontend/src/hooks/useMaintenance.ts

```diff
@@ -84,6 +84,14 @@ enum States {
   ZERO_REDEMPTIONS = 'zeroRedemptions',
   ZERO_DLLR = 'zeroDLLR',
   ZERO_EXPORT_CSV = 'zeroExportCSV',
+
+  STAKING_FULL = 'stakingFull',
+  STAKING_STAKE_SOV = 'stakingStakeSOV',
+  STAKING_NEW = 'stakingNew',
+  STAKING_INCREASE = 'stakingIncrease',
+  STAKING_EXTEND = 'stakingExtend',
+  STAKING_DECREASE = 'stakingDecrease',
+  STAKING_DELEGATE = 'stakingVestingDelegate',
```

Comment:
`STAKING_STAKE_SOV` is not used, it should prevent users from creating/increasing/extending their stakes

similar thing for the others as well (only `STAKING_FULL` and `STAKING_NEW` are implemented)